tests/unit_tests/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ function(gridtools_add_unit_test tgt)
     if(NOT ARGS_NO_NVCC AND TARGET _gridtools_nvcc AND NOT nvcc_compilation_checked)
         _gt_convert_to_cuda_source(sources ${ARGS_SOURCES})
         add_library(${tgt}_nvcc OBJECT ${sources})
+        target_compile_options(${tgt}_nvcc PRIVATE $<$<COMPILE_LANGUAGE:CUDA>:--expt-relaxed-constexpr>)
```

Comment:
I think this should not be here. It should come from the gridtools_setup_targets.cmake:177.

Reply:
it does not. because this target does not have any library dependencies. And it should not. -- It just checks compilation. No binary is produced. only obj file that is dropped on the floor

Reply:
Got it. Can you add a description to `gridtools_add_unit_test()`. Or I can add it in a separate PR. It took me a moment to understand the pattern...

Reply:
done